spec/integration/machinery_opensuse13.1_spec.rb

```diff
@@ -17,7 +17,7 @@
 
 require_relative "integration_spec_helper"
 
-describe "machinery@openSUSE 13.1" do
+describe "machinery@openSUSE 13.1", host: "openSUSE_13_1" do
```

Comment:
Would it be possible to get rid of the extra argument and extract the host from the example name?